### ISSUE TITLE
[rllib] torch - This should be logits=inputs

### ISSUE THREAD

**Issue by royf** [ray-project/ray#8175]
https://github.com/ray-project/ray/blob/e9ee5c4e5f27c5fa547f1c5c504b9efebef21026/rllib/models/torch/torch_action_dist.py#L63
